<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="5" passed="2" failed="1" skipped="2">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-06-12T05:11:31Z" name="Suite" finished-at="2020-06-12T05:12:59Z" duration-ms="88014">
    <groups>
      <group name="Regression">
        <method signature="LoginPageTest.ToverifyLoginFunctionality()[pri:2, instance:com.amazon.qa.testcases.LoginPageTest@d83da2e]" name="ToverifyLoginFunctionality" class="com.amazon.qa.testcases.LoginPageTest"/>
        <method signature="HomePageTest.verifyusername()[pri:1, instance:com.amazon.qa.testcases.HomePageTest@2805c96b]" name="verifyusername" class="com.amazon.qa.testcases.HomePageTest"/>
        <method signature="HomePageTest.Navigatetobook()[pri:3, instance:com.amazon.qa.testcases.HomePageTest@2805c96b]" name="Navigatetobook" class="com.amazon.qa.testcases.HomePageTest"/>
      </group> <!-- Regression -->
    </groups>
    <test started-at="2020-06-12T05:11:31Z" name="To Check login Class Functionality" finished-at="2020-06-12T05:12:27Z" duration-ms="56230">
      <class name="com.amazon.qa.testcases.LoginPageTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@d83da2e]" started-at="2020-06-12T10:41:31Z" name="setup" finished-at="2020-06-12T10:41:47Z" duration-ms="16322" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="ToVerifyThePageTitle()[pri:1, instance:com.amazon.qa.testcases.LoginPageTest@d83da2e]" started-at="2020-06-12T10:41:47Z" name="ToVerifyThePageTitle" finished-at="2020-06-12T10:41:47Z" duration-ms="58" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ToVerifyThePageTitle -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@d83da2e]" started-at="2020-06-12T10:41:47Z" name="close" finished-at="2020-06-12T10:41:55Z" duration-ms="7402" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@d83da2e]" started-at="2020-06-12T10:41:55Z" name="setup" finished-at="2020-06-12T10:42:12Z" duration-ms="17058" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="ToverifyLoginFunctionality()[pri:2, instance:com.amazon.qa.testcases.LoginPageTest@d83da2e]" started-at="2020-06-12T10:42:12Z" name="ToverifyLoginFunctionality" finished-at="2020-06-12T10:42:17Z" duration-ms="5652" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ToverifyLoginFunctionality -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.amazon.qa.testcases.LoginPageTest@d83da2e]" started-at="2020-06-12T10:42:17Z" name="close" finished-at="2020-06-12T10:42:27Z" duration-ms="9399" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.amazon.qa.testcases.LoginPageTest -->
    </test> <!-- To Check login Class Functionality -->
    <test started-at="2020-06-12T05:12:28Z" name="To check HomePage Class Functionality" finished-at="2020-06-12T05:12:59Z" duration-ms="30679">
      <class name="com.amazon.qa.testcases.HomePageTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.amazon.qa.testcases.HomePageTest@2805c96b]" started-at="2020-06-12T10:42:28Z" name="setup" finished-at="2020-06-12T10:42:48Z" duration-ms="19654" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyusername()[pri:1, instance:com.amazon.qa.testcases.HomePageTest@2805c96b]" started-at="2020-06-12T10:42:48Z" name="verifyusername" finished-at="2020-06-12T10:42:48Z" duration-ms="149" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Hello, Deepika] but found [Account & Lists]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Hello, Deepika] but found [Account & Lists]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.amazon.qa.testcases.HomePageTest.verifyusername(HomePageTest.java:50)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyusername -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.amazon.qa.testcases.HomePageTest@2805c96b]" started-at="2020-06-12T10:42:48Z" name="teardown" finished-at="2020-06-12T10:42:59Z" duration-ms="10090" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method signature="Navigatetobook()[pri:3, instance:com.amazon.qa.testcases.HomePageTest@2805c96b]" started-at="2020-06-12T10:42:59Z" name="Navigatetobook" finished-at="2020-06-12T10:42:59Z" depends-on-methods="com.amazon.qa.testcases.HomePageTest.verifyusername" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method HomePageTest.Navigatetobook()[pri:3, instance:com.amazon.qa.testcases.HomePageTest@2805c96b] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method HomePageTest.Navigatetobook()[pri:3, instance:com.amazon.qa.testcases.HomePageTest@2805c96b] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1039)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Navigatetobook -->
        <test-method signature="click_on_CC()[pri:4, instance:com.amazon.qa.testcases.HomePageTest@2805c96b]" started-at="2020-06-12T10:42:59Z" name="click_on_CC" finished-at="2020-06-12T10:42:59Z" depends-on-methods="com.amazon.qa.testcases.HomePageTest.verifyusername" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method HomePageTest.click_on_CC()[pri:4, instance:com.amazon.qa.testcases.HomePageTest@2805c96b] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method HomePageTest.click_on_CC()[pri:4, instance:com.amazon.qa.testcases.HomePageTest@2805c96b] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1039)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- click_on_CC -->
      </class> <!-- com.amazon.qa.testcases.HomePageTest -->
    </test> <!-- To check HomePage Class Functionality -->
  </suite> <!-- Suite -->
</testng-results>
